tests/Functional/Platform/ConcatTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Doctrine\DBAL\Tests\Functional\Platform;
+
+use Doctrine\DBAL\Platforms\MySQLPlatform;
+use Doctrine\DBAL\Platforms\SQLServer2012Platform;
+use Doctrine\DBAL\Schema\Table;
+use Doctrine\DBAL\Tests\FunctionalTestCase;
+
+class ConcatTest extends FunctionalTestCase
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        if ($this->connection->getDatabasePlatform() instanceof MySQLPlatform) {
+            return;
+        }
+
+        if ($this->connection->getDatabasePlatform() instanceof SQLServer2012Platform) {
+            return;
+        }
+
+        self::markTestSkipped('Restricted to MySQL and SQL Server.');
+    }
+
+    public function testConcat(): void
+    {
+        $table = new Table('concat_test');
+        $table->addColumn('name', 'string');
+        $this->connection->getSchemaManager()->dropAndCreateTable($table);
```

Comment:
Why is it necessary? Concatenation in SQL can be tested without a table. Please see the example I pointed to earlier.